internal/util/pkgutil/pkgutil.go

```diff
@@ -343,10 +333,6 @@ func RoundTripKptfilesInPkg(pkgPath string) error {
 
 // Exists returns true if a file or directory exists on the provided path,
 // and false otherwise.
-func Exists(path string) (bool, error) {
-	_, err := os.Stat(path)
-	if err != nil && !os.IsNotExist(err) {
-		return false, err
-	}
-	return !os.IsNotExist(err), nil
+func Exists(path types.FileSystemPath) (bool, error) {
```

Comment:
Can this just return bool instead ?

Reply:
Possibly? Trying to minimize changes the only difference from before would be the `path string` argument

https://github.com/GoogleContainerTools/kpt/blob/5169954be82fabcfb6f7c0a01f4803c81cb65c67/internal/util/pkgutil/pkgutil.go#L343-L351

That would probably be better as a new issue to change, rather than changing as part of the `string`->`FileSystemPath` replacement